website/static/css/custom.css

```diff
@@ -340,8 +357,19 @@ span.arrow::after {
         padding-bottom: 8px;
         background-color: rgba(255, 254, 254, 0.2);
```

Comment:
In general, I recommend we stick with either hex or rgb, and preferably create and reference variables for all colors. I tend to use hex over rgb. CSS supports variables but my pov is that SASS still does it best. In this case you need less opacity (rgba) and it makes sense to accomplish with rgba vs another line of opacity.

Reply:
converted all other colors to hex and added CSS variables throughout

left `rgba` instances alone due to opacity

Reply:
✅